src/Microsoft.TemplateEngine.Edge/TemplateUpdates/DefaultInstallUnitDescriptor.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Abstractions.TemplateUpdates;
+
+namespace Microsoft.TemplateEngine.Edge.TemplateUpdates
+{
+    public class DefaultInstallUnitDescriptor : IInstallUnitDescriptor
+    {
+        public DefaultInstallUnitDescriptor(Guid descriptorId, Guid mountPointId, string identifier)
+        {
+            DescriptorId = descriptorId;
+            MountPointId = mountPointId;
+            Identifier = identifier;
+            Details = new Dictionary<string, string>();
+            DetailKeysDisplayOrder = new List<string>();
```

Comment:
Same with the possibility of sharing this one - might be able to use `Empty<T>` as well